eve/workers/openstack-single-node/requirements.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+yum install -y curl \
```

Comment:
Trailing backslash

Reply:
Yup I did that on purpose so we can add dependencies to this list without creating unnecessary diffs when adding the backslash at that time. I can remove it though, I have absolutely no preference in terms of style guides in Shell.

Reply:
If you want to achieve this, you should rather create an array containing package names, this way the diff will be ok.
e.g.:
```
declare -a PACKAGES=(
    curl
)
```

Reply:
Oh that looks good! Thanks for the tip, let me use it :)